frontend/index.html

```diff
@@ -1,16 +1,14 @@
 <!DOCTYPE html>
-<html lang="">
+<html lang="" data-tbpro-service="apmt">
```

Comment:
Used in the css to select the correct service colours. We could also use another solution but I had the `.dark` class on the brain. Alternatively you can use "send" as the value to get send's colours.

Reply:
cc @aaspinwall for the future where send has a ui and I've actually split this off into its own repo 😅